python/ray/tests/test_scheduling.py

```diff
@@ -519,6 +519,64 @@ def f(s, ref):
     wait_for_condition(lambda: not fetches_queued())
 
 
+def build_cluster(num_cpu_nodes, num_gpu_nodes):
+    cluster = ray.cluster_utils.Cluster()
+    gpu_ids = [
+        cluster.add_node(num_cpus=10, num_gpus=1).unique_id
+        for _ in range(num_gpu_nodes)
+    ]
+    cpu_ids = [
+        cluster.add_node(num_cpus=1).unique_id for _ in range(num_cpu_nodes)
+    ]
+    cluster.wait_for_nodes()
+    return cluster, cpu_ids, gpu_ids
+
+
+def test_gpu(monkeypatch):
+    monkeypatch.setenv("RAY_scheduler_avoid_gpu_nodes", "1")
```

Comment:
You can pass this in _system_config to ray init instead when adding the node instead

Reply:
noted.
maybe we can keep this as is (its very explicit, toplevel. and short).